<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SampleListDialog</class>
 <widget class="QDialog" name="SampleListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>样本列表</string>
  </property>
 <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="filterLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="projectLabel">
       <property name="text">
        <string>项目</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="projectCombo"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="batchLabel">
       <property name="text">
        <string>批次</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QComboBox" name="batchCombo"/>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="sampleLabel">
       <property name="text">
        <string>样本</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QComboBox" name="sampleCombo">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="searchLabel">
       <property name="text">
        <string>搜索</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QLineEdit" name="searchEdit">
       <property name="placeholderText">
        <string>输入关键字（项目/批次/短码/平行号）</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="bigTab">
      <attribute name="title">
       <string>大热重</string>
      </attribute>
      <layout class="QVBoxLayout" name="bigLayout">
       <item>
        <widget class="QTableView" name="bigTableView"/>
       </item>
      </layout>
     </widget>
    <widget class="QWidget" name="smallTab">
     <attribute name="title">
      <string>小热重</string>
     </attribute>
     <layout class="QVBoxLayout" name="smallLayout">
      <item>
       <widget class="QTableView" name="smallTableView"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="smallRawTab">
     <attribute name="title">
      <string>小热重（原始数据）</string>
     </attribute>
     <layout class="QVBoxLayout" name="smallRawLayout">
      <item>
       <widget class="QTableView" name="smallRawTableView"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="chromTab">
     <attribute name="title">
      <string>色谱</string>
     </attribute>
      <layout class="QVBoxLayout" name="chromLayout">
       <item>
        <widget class="QTableView" name="chromTableView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="processBigTab">
      <attribute name="title">
       <string>工序大热重</string>
      </attribute>
      <layout class="QVBoxLayout" name="processBigLayout">
       <item>
        <widget class="QTableView" name="processBigTableView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="previewButton">
       <property name="text">
        <string>预览</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>SampleListDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>750</x>
     <y>530</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
